components/user/themes/UserTabs.js

```diff
@@ -46,16 +46,48 @@ export default function UserTabs({ data, BASE_URL }) {
   });
   const [tabs, setTabs] = useState(displayTabs);
   const changeTab = (e, value) => {
-    e.preventDefault();
+    e.preventDefault();    
     setTabs(
-      tabs.map((tab) =>
-        tab.name === e.target?.value || tab.name === value
-          ? { ...tab, current: true }
-          : { ...tab, current: false }
+      tabs.map((tab) =>{
+        if (tab.name === e.target?.value || tab.name === value){
+        addTabsToURL(tab.url)
+          return { ...tab, current: true }
+        } else {
+          return { ...tab, current: false }}
+        }
       )
     );
   };
 
+  useEffect(() => {
+    const selectedTab = tabs.map((tab) =>
+        (tab.url === getTabsURL() ) ?
+          { ...tab, current: true }
+          :
+          { ...tab, current: false }
+        
+      )
+    
+      setTabs(selectedTab)
+  }, [])
```

Comment:
That's why it's not included, I considered it and tried it but the useEffect should only run on the first render. Otherwise it causes a problem, because it updates the tabs only when the link is loaded, after that it shouldn't do anything else, now the tabs should change on any other tab button's click👌🏽